cEP-2.md

```diff
@@ -0,0 +1,33 @@
+#Next Gen Bear Design
+| Metadata     | 					                        |
+| ------------ |-----------------------------------------|
+| cEP          | 2                                       |
+| Version      | 2.1                                     |  
```

Comment:
How are these versions done. I thought because this is the 1st version of cEP-2 that is why 2.1?